demo/demo.pinery_ius-input

```diff
@@ -1206,6 +1353,14 @@
     "dv200": null,
     "external_donor_id": "ABC_0001",
     "external_tissue_id": null,
+    "flowcell_geometry": [
```

Comment:
Will this ever be, say, [1, 4, 2, 3]? I'm confused why it is this format rather than an int length

Reply:
No. This format is kind of awkward, but...ughhh. So, if there are 4 compartments, then it could be that everything is joined:
`[[1, 2, 3, 4]]` or everything is split `[[1], [2], [3], [4]]`

There's a possibility that Illumina makes a half-joined flowcell, like `[[1, 2], [3, 4]]`, but I don't think that has happened.